aspnetcore/mvc/views/view-compilation.md

```diff
@@ -21,6 +21,9 @@ Build-time and publish-time compilation of Razor files is enabled by default by
 
 In addition to build-time compilation, updating Razor views and Razor Pages is supported using <xref:test/hot-reload>.
 
+> [!NOTE]
+> When enabled, runtime compilation currently disables [.NET Hot Reload](xref:test/hot-reload).
```

Comment:
```suggestion
> When enabled, runtime compilation currently disables [.NET Hot Reload](xref:test/hot-reload). Runtime compilation with Hot Reload is planned for a future release.
```

Reply:
I like your suggestion but that's much stronger that what I'm saying. @danroth27 will need to approve.

Reply:
Adding that I assume (obviously) that it would still be desirable for some scenarios to have both and there might be an intention one day to make them happy together ... per https://github.com/dotnet/AspNetCore.Docs/issues/23910#issuecomment-1033107013 ...

A dev ...

> is Runtime compilation only today in .NET 6 applicable if I want to do things on the server once my app is deployed?

David ...

> That's the intent yes. Only if you need to change the razor at runtime after deployment would you need runtime compilation.

There's an open issue to add something on that point to the runtime compilation and/or Hot Reload guidance at ...

https://github.com/dotnet/AspNetCore.Docs/issues/24944